src/keycloak/chart/templates/istio-admin.yaml

```diff
@@ -8,9 +8,10 @@ metadata:
   name: keycloak-block-admin-access-from-public-gateway
   namespace: {{ .Release.Namespace }}
 spec:
-  selector:
-    matchLabels:
-      {{- include "keycloak.selectorLabels" . | nindent 6 }}
+  targetRefs:
+    - kind: Service
+      group: ""
+      name: "keycloak-http"
```

Comment:
Mostly noting for myself - it seems like [this is valid](https://istio.io/latest/docs/reference/config/security/authorization-policy/#AuthorizationPolicy) although I think I've seen more examples of targetting the Waypoint itself directly, rather than the service. I assume the difference here is more important if you have a single waypoint for a namespace, with multiple services - this would give you the granularity to target only a single service, rather than the full namespace.

Reply:
This seems to work fine. Thanks for suggestion!